java/src/main/java/com/google/cloud/dataproc/templates/databases/CassandraToBQ.java

```diff
@@ -80,5 +82,24 @@ public void runTemplate() throws StreamingQueryException, TimeoutException {
         .save();
   }
 
-  public void validateInput() {}
+  public void validateInput() {
+    if (StringUtils.isAllBlank(config.getHost())
+        || StringUtils.isAllBlank(config.getBqLocation())
+        || StringUtils.isAllBlank(config.getTemplocation())) {
```

Comment:
Cassandra to BQ already has Jakarta validations in place using the Config class. We need not have to perform validations again.

Reply:
I thought the same, but the issue was created to add validation code in the validateInput method.
Should I close all the validateInput issues on templates which already have Jakarta validations?
(There are a few more open issues for Spanner to GCS, Redshift to GCS etc)

Reply:
Actually @saumyasinha-google , the idea was to keep the Jakarta validation framework and the calling of this framework should be done via validateInput method. So that ways, both the agenda would be fulfilled

1. We would be having Jakarta validation framework in our templates.
2. We would be having validateInput method, to check for validations.

Reply:
Added Jakarta validations in validateInput method.